cpp/oneapi/dal/algo/pca/backend/cpu/train_kernel_svd.cpp

```diff
@@ -61,21 +69,23 @@ static result_t call_daal_kernel(const context_cpu& ctx,
     const auto daal_data =
         interop::convert_to_daal_homogen_table(arr_data, row_count, column_count);
     const auto daal_eigenvectors =
-        interop::convert_to_daal_homogen_table(arr_eigvec, column_count, component_count);
+        interop::convert_to_daal_homogen_table(arr_eigvec, component_count, column_count);
     const auto daal_eigenvalues =
         interop::convert_to_daal_homogen_table(arr_eigval, 1, component_count);
     const auto daal_means = interop::convert_to_daal_homogen_table(arr_means, 1, column_count);
     const auto daal_variances = interop::convert_to_daal_homogen_table(arr_vars, 1, column_count);
 
     daal_pca::internal::InputDataType dtype = daal_pca::internal::nonNormalizedDataset;
 
-    daal_pca::interface3::BatchParameter<Float, daal_pca::svdDense> parameter;
+    auto norm_alg = get_normalization_algorithm<Float>();
+    norm_alg->input.set(daal::algorithms::normalization::zscore::data, daal_data);
+    norm_alg->parameter().resultsToCompute |= daal::algorithms::normalization::zscore::mean;
+    norm_alg->parameter().resultsToCompute |= daal::algorithms::normalization::zscore::variance;
```

Comment:
```suggestion
    namespace daal_zscore = daal::algorithms::normalization::zscore;
    
    auto norm_alg = get_normalization_algorithm<Float>();
    norm_alg->input.set(daal_zscore::data, daal_data);
    norm_alg->parameter().resultsToCompute |= daal_zscore::mean;
    norm_alg->parameter().resultsToCompute |= daal_zscore::variance;
```
just a nip; optional to apply